cpp/CMakeLists.txt

```diff
@@ -420,6 +420,25 @@ if(BUILD_CUML_CPP_LIBRARY)
         src/hdbscan/hdbscan.cu
         src/hdbscan/condensed_hierarchy.cu
         src/hdbscan/prediction_data.cu)
+
+    set_property(
```

Comment:
@divyegala We need a comment about why these are here. Can you add a link to an issue everywhere that this workaround was used?
```suggestion
    # When using GCC 13, some maybe-uninitialized warnings appear from CCCL and are treated as errors.
    # See this CCCL issue: <INSERT LINK HERE>
    set_property(
```